coverport/coverage_server.py

```
@@ -0,0 +1,54 @@
+            except Exception as e:
+                self.send_response(500)
+                self.send_header("Content-Type", "text/plain")
+                self.end_headers()
+                self.wfile.write(f"Error: {e}".encode())
+                return
+        self.send_response(200)
+        self.send_header("Content-Type", "text/plain")
+        self.end_headers()
+        self.wfile.write(b"CoverPort coverage server ready")
+
+    def log_message(self, format, *args):
```

Ruff (0.15.12)

[error] Function argument `format` is shadowing a Python builtin

(A002)